modules/heat_conduction/doc/content/source/bcs/GaussianEnergyFluxBC.md

```diff
@@ -0,0 +1,23 @@
+# GaussiandEnergyFluxBC
+
+!syntax description /BCs/GaussiandEnergyFluxBC
+
+This boundary condition computes an influx of energy from a beam (e.g. laser) with a
+Gaussian spatial profile. The flux is given by
+
+\begin{equation}
+-2r_{eff}F_0\exp{-r_{eff}r_p^2/R^2}
+\end{equation}
+
+where $r_eff$ is an effective radius used to specify the radial distribution of
+beam energy, $F_0$ is the average heat flux of the beam, $R$ is the beam radius,
+and $r_p$ is the normed distance from the point at which we're evaluating the
+flux to the centerpoint of the beam. This functional form of the beam flux is
+taken from [!cite](noble2007use). The negative sign on the flux indicates that
+the flux is incoming.
+
```

Comment:
as a help to the user I would add the integral of the flux assuming all the beam is hitting a flat surface on the domain.

Reply:
Added a sentence. See if it addresses what you were saying